cmd/s3.go

```diff
@@ -52,6 +53,12 @@ func (a *MethodAws) InitS3Command() {
 		Use:   "externalenumerate",
 		Short: "Enumerate a single public facing S3 bucket.",
 		Long:  `Enumerate a single public facing S3 bucket with no credentials.`,
+		PersistentPreRunE: func(cmd *cobra.Command, _ []string) error {
+			outputFormat, _ := cmd.Flags().GetString("output")
+			outputFile, _ := cmd.Flags().GetString("output-file")
+			authed := false
+			return a.setupCommonConfig(cmd, outputFormat, outputFile, authed)
```

Comment:
Some comment about boolean. Also, do we want to check the errs on grabbing the flags?